src/util-decode-mime.h

```diff
@@ -96,6 +97,9 @@ typedef struct MimeDecConfig {
     int decode_base64;  /**< Decode base64 bodies */
     int decode_quoted_printable;  /**< Decode quoted-printable bodies */
     int extract_urls;  /**< Extract and store URLs in data structure */
+    ConfNode *extract_urls_schemes; /**< List of schemes of which to
+                                         extract urls  */
+    int log_url_scheme;             /**< Log the scheme of extracted URLs */
```

Comment:
can you switch this to `bool`? I see other existing code can be improved with that as well, if you're interested (in a separate commit)

Reply:
Sure, will do!